Runtime/Scripts/MobiledgeXIntegrationHelper.cs

```diff
@@ -324,21 +325,6 @@ public async Task<bool> IsRoaming()
     }
 #endif
 
-#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
-    public async Task<Dictionary<string, string>> GetDeviceInfo()
-#pragma warning restore CS1998 // Async method lacks 'await' operators and will run synchronously
-    {
-      var deviceInfo = matchingEngine.deviceInfo.GetDeviceInfo();
-#if UNITY_IOS
-      if (deviceInfo == null || deviceInfo.Count == 0) 
-      {
-        return new Dictionary<string, string>();
-      }
-      UpdateLocationFromDevice();
-      deviceInfo["NetworkCountryIso"] = await carrierInfoClass.ConvertGPSToISOCountryCode(location.longitude, location.latitude);
-#endif
-      return deviceInfo;
-    }
 
     /// <summary>
     /// Checks whether the default netowrk data path Edge is Enabled on the device or not, Edge requires connections to run over cellular interface.
```

Comment:
Nit: netowrk -> network

Reply:
Fixed in https://github.com/mobiledgex/edge-cloud-sdk-unity/pull/174/commits/f8c0f09719433b383b86a3b2bcef10f04cfa8c8b